packages/dapp/src/components/listing/ListingDiscourse.tsx

```diff
@@ -0,0 +1,34 @@
+import * as React from "react";
+import * as ReactDOM from "react-dom";
+
+import { ViewModule } from "../utility/ViewModules";
+
+class ListingDiscourse extends React.Component {
+  constructor(props: {}) {
+    super(props);
+  }
+
+  public async componentDidMount(): Promise<void> {
+    const discourseUrl = process.env.REACT_APP_DISCOURSE_URL;
+    const tag = document.createElement("script");
+    tag.async = true;
+    tag.src = discourseUrl + "/static/embed.js";
+
+    const handleCallback = () => {
```

Comment:
how does this get cleaned up, how does it load correct discussion

Reply:
It loads it in an iframe and creates a discussion per URL